### ISSUE TITLE
Regularly running projects through HTML validator/a11y checkers?

### ISSUE THREAD

**Issue by iteles** [foundersandcoders/master-reference#81]
We talk about valid HTML and accessibility (a11y) in week one (cc @Cleop @jsms90 @njsfield), but my experience of it is that after this, it entirely disappears.

What does everyone think about part of the Friday presentations on a regular basis include running the project through:
+ An HTML validator (e.g. https://validator.w3.org/nu/)
+ An accessibility checker (e.g. http://wave.webaim.org/ or http://www.tenon.io)

**Comment by njsfield** [comment]
Agreed. The [project.md write up for Week 1](https://github.com/foundersandcoders/master-reference/blob/week-1-toolkit/coursebook/week-1/project.md) outlines accessibility based user stories, with a note that 
> We will also be checking your site for accessibility on Tenon.io

Would be cool if other projects had accessibility requirements

**Comment by iteles** [comment]
@njsfield Who needs to see this issue in order to make that happen? :tada:

**Comment by njsfield** [comment]
(I can't assign any more people but hopefully @esraajb @stevehopkinson @shiryz sees this)...

**Comment by njsfield** [comment]
Thumbs up for projects run through accessibility validators on friday through [HTML validator](https://validator.w3.org/nu/) and [tenon.io](http://www.tenon.io)

**Comment by njsfield** [comment]
Thumbs up for projects not run through accessibility validators on friday

**Comment by jsms90** [comment]
@iteles @njsfield I haven't looked at http://wave.webaim.org/ or http://www.tenon.io yet. But from what I have seen of https://validator.w3.org/nu/, I much prefer using [this "Accessibility devloper tools" chrome extension](https://chrome.google.com/webstore/detail/accessibility-developer-t/fpkknkljclfencbdbgkenhalefipecmb?hl=en) and [this "HTML5 Outliner" chrome extension](https://chrome.google.com/webstore/detail/html5-outliner/afoibpobokebhgfnknfndkgemglggomo?hl=en) are better tools, because you don't need to go to a separate website and copy in the link to your website. A chrome extension allows you to stay on the site you are concerned with.

**Comment by jsms90** [comment]
HTML5 Outliner is the lightest tool ever - just a quick easy way of seeing how semantic and well-ordered your DOM elements are

tenon.io and wave actually look very thoroughly, but I still prefer dev tools

If anyone finds any more, please raise an issue and I'll update [the list](https://github.com/jsms90/web-accessibility/blob/master/putting-yourself-in-someone-elses-shoes.md#tools-that-can-help)
